dot/types/inherents.go

```diff
@@ -4,90 +4,78 @@
 package types
 
 import (
-	"bytes"
-	"encoding/binary"
-	"errors"
 	"fmt"
-	"math/big"
 
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
-var (
-	// Timstap0 is an inherent key.
-	Timstap0 = []byte("timstap0")
-	// Babeslot is an inherent key.
-	Babeslot = []byte("babeslot")
-	// Uncles00 is an inherent key.
-	Uncles00 = []byte("uncles00")
+// InherentIdentifier is an identifier for an inherent.
+type InherentIdentifier uint
+
+const (
+	// Timstap0 is the identifier for the `timestamp` inherent.
+	Timstap0 InherentIdentifier = iota
+	// Babeslot is the BABE inherent identifier.
+	Babeslot
+	// Uncles00 is the identifier for the `uncles` inherent.
+	Uncles00
+	// Parachn0 is an inherent key for parachains inherent.
+	Parachn0
+	// Newheads is an inherent key for new minimally-attested parachain heads.
+	Newheads
 )
 
+// Bytes returns a byte array of given inherent identifier.
+func (ii InherentIdentifier) Bytes() [8]byte {
+
+	kb := [8]byte{}
+	switch ii {
+	case Timstap0:
+		copy(kb[:], []byte("timstap0"))
+	case Babeslot:
+		copy(kb[:], []byte("babeslot"))
+	case Uncles00:
+		copy(kb[:], []byte("uncles00"))
+	case Parachn0:
+		copy(kb[:], []byte("parachn0"))
+	case Newheads:
+		copy(kb[:], []byte("newheads"))
```

Comment:
nit why is it named `kb`? Shouldn't it be `inherentIndentifier` / `identifier` / `identifierBytes`